Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -43,6 +39,11 @@ if($deployFilesInParallel -eq "false" -or  ( $resources.Count -eq 1 ) )
 		$resOperationId = Invoke-ResourceOperation -EnvironmentName $environmentName -ResourceName $machine -EnvironmentOperationId $envOperationId -Connection $connection -ErrorAction Stop
 		Write-Verbose "ResourceOperationId = $resOperationId" -Verbose
 		
+		$machineUserName = $resource.Username
```

Comment:
Can you convert tabs to spaces in this file, it has really affected formatting of code. Looks untidy.